website/releases/release-0.12.0.md

```diff
@@ -9,6 +9,61 @@ last_modified_at: 2022-08-17T10:30:00+05:30
 
 ## Release Highlights
 
+### Migration Guide
```

Comment:
Should the `Migration Guide` stand as a top-level section parallel to `Release Highlights`like 0.10.0: https://hudi.apache.org/releases/release-0.10.0?